releases/links_v2.0.0/src/main.cpp

```diff
@@ -0,0 +1,43 @@
+#include "LINKS.hpp"
+#include "InputParser.hpp"
+#include <chrono>
+#include <stdlib.h> // for sleep
+
+int main(int argc, char** argv){
+    InputParser* linksArgParser = new InputParser(argc, argv);
```

Comment:
Not necessary for this release, but it'd be good to learn to use smart pointers in C++. Also, it doesn't seem like you need pointers here, anyway. Why not create the object on stack?

Reply:
I paid attention to implement with smart pointers in `LINKS.hpp` though you're right this should not be a dynamic object